app/tests/spec/views/settings/account_recovery/recovery_key.js

```diff
@@ -0,0 +1,166 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import $ from 'jquery';
+import {assert} from 'chai';
+import Broker from 'models/auth_brokers/base';
+import Metrics from 'lib/metrics';
+import {Model} from 'backbone';
+import Notifier from 'lib/channels/notifier';
+import Relier from 'models/reliers/base';
+import SentryMetrics from 'lib/sentry';
+import sinon from 'sinon';
+import TestHelpers from '../../../../lib/helpers';
+import User from 'models/user';
+import View from 'views/settings/account_recovery/recovery_key';
+import WindowMock from '../../../../mocks/window';
+
+describe('views/settings/account_recovery/recovery_key', () => {
+  let account, broker, email, metrics, notifier, relier, sentryMetrics, user, view, model, windowMock, keyExists;
+  const UID = '123', recoveryKey = 'AV4TY1EXMEQARDBQ';
+
+  function initView() {
+    windowMock = new WindowMock();
+    windowMock.document = {
+      body: {},
+      execCommand: () => {},
+      getElementById: () => {}
+    };
+
+    view = new View({
+      broker,
+      metrics,
+      model,
+      notifier,
+      relier,
+      user,
+      window: windowMock
+    });
+
+    sinon.stub(view, 'getSignedInAccount').callsFake(() => account);
+
+    return view.render()
+      .then(() => $('#container').html(view.$el));
+  }
+
+  beforeEach(() => {
+    broker = new Broker();
+    email = TestHelpers.createEmail();
+    notifier = new Notifier();
+    sentryMetrics = new SentryMetrics();
```

Comment:
is sentryMetrics actually used anywhere? If not, you could just pass in a mock? (Applies to the other tests too)